packages/workbox-background-sync/src/lib/request-queue.js

```diff
@@ -96,6 +96,7 @@ class RequestQueue {
    * @private
    */
   async push({request}) {
+    await this._initializationPromise;
```

Comment:
my bad, i'll add it to `saveQueue` as well.
Also would adding it to saveQueue be enough? where would we want to use an async getter?